edx_proctoring/tests/test_api.py

```diff
@@ -1,12 +1,10 @@
 # coding=utf-8
```

Comment:
Reviewers: GitHub is making this look complicated, but all I did is moved a number of tests out into two new test files, to make this file more manageable.